app/transport/rest/user.go

```diff
@@ -97,12 +122,37 @@ func (uh UserHandler) GetUserByID() http.Handler {
 		}
 
 		user := User{
-			ID:        entUser.ID,
-			FirstName: entUser.FirstName,
-			LastName:  entUser.LastName,
-			Email:     entUser.Email,
-			CreatedAt: entUser.CreatedAt,
+			ID:        solidUser.ID,
+			FirstName: solidUser.FirstName,
+			LastName:  solidUser.LastName,
+			Email:     solidUser.Email,
+			CreatedAt: solidUser.CreatedAt,
 		}
 		WriteJSONResponse(w, http.StatusOK, user, uh.logger)
 	})
 }
+
+// GetUsers retrieves all entities.User by given parameters.
+func (uh UserHandler) GetUsers() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		var params = map[string]string{
+			sort:   firstName + "," + lastName,
```

Comment:
It's not good that you are using constants here that know DB table column names...
You can pass any value here, like `firstName` and on repository level to do a mapping between `firstName` and corresponding column name in DB.

That's needed cos only repository should know DB details (like column names etc).

Do, it's good that you have constants here but you must map their values in repository level as well.

Reply:
Got it, Thanks!